### ISSUE TITLE
Directions for Sub-Zero for Plex Media Server on NVidia Shield

### ISSUE THREAD

**Issue by aagha** [pannal/Sub-Zero.bundle#606]
Is there a good place to find directions for how to install Sub-Zero for PMS on the NVidia Shield?

**Comment by pannal** [comment]
Not really. Since the channels are gone from official clients, the only option is to do a manual install, which requires root AFAIK.

**Comment by rbmaster** [comment]
Go to your Plex settings on the ShieldTV and move the data to the user accessible area, then upload the the sub-zero bundle to the plugin folder (now accessible) and finally go to the Plex settings again and move the data back to the system storage (so the scripts inside the bundle have permission to run)

I wouldn't call this a solution, it's more like a workaround, but it works.
